README.md

```diff
@@ -1960,6 +1960,7 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 - [Kutt](https://kutt.it) - A modern URL shortener with support for custom domains. ([Source Code](https://github.com/thedevs-network/kutt)) `MIT` `Nodejs`
 - [liteshort](hhttps://git.ikl.sh/132ikl/liteshort) - User-friendly, actually lightweight, and configurable URL shortener. `MIT` `Python`
 - [Lstu](https://github.com/ldidry/lstu) - Lightweight URL shortener. `WTFPL` `Perl`
+- [RabiJump](https://github.com/KernelErr/RabiJump) - A simple and efficient URL shortener with web panel. `Apache-2.0` `Docker`
```

Comment:
The [Dockerfile](https://github.com/KernelErr/RabiJump/blob/main/server/Dockerfile) uses `FROM mcr.microsoft.com/cbl-mariner/distroless/base:2.0`. Where can I find the source code for this image?

Reply:
Hi @nodiscc, it's a distroless image from Microsoft. And it's not open source yet I think. If needed, I could change it to other open source distroless images. :)

Reply:
I use this as it's a clean minimal runtime. There is a repository for CBL-Mariner: https://github.com/microsoft/CBL-Mariner.

Reply:
> the image is not open source yet I think

Then RabiJump must be added to `non-free.md` instead.

> I could change it to other open source distroless images

It would be nice.